### PR TITLE
clean up summer meeting notes

### PR DESCRIPTION
**Pull request by paulshriner**
I cleaned up the 2nd through 6th summer meeting notes a bit. The main issue I noticed is that they were basically a block of text, which is not the easiest to read and could deter someone from reading the notes. I resolved this by breaking up the large chunks of text into smaller sections, as well as using bulleted lists when a list seemed appropriate (the entire notes should not be a bulleted list). 

There were also a few issues of clarity which I attempted to solve (overall, the content was well covered, but a few places did not line up with what actually happened during the meeting). 

I want us to discuss this more (in a future meeting?), but the general gist is that these notes should be a way for people to easily understand what we discussed in meetings. They should not be more lengthy or detailed than needed, and should be clear and easy to understand.

I updated Hugo to 0.127.0.

### CODE REVIEW

**Comment by mwilliams52000** [comment]
I generally agree with the changes made here/the pivot to using a brisker note-taking style, and am fine with this pull request being merged

**Comment by paulshriner** [comment]
> generally

It sounds like there is some hesitation here. I want everyone to understand the changes and why they were made. If you have a concern, no matter how small or nitpicky, bring it up (others feel free to chime in)!

I get that most people might not even care about the meeting notes but they still do represent us, and are great for people who want to find out what we're doing.

**Comment by mwilliams52000** [comment]
I don't have any hesitations about the content of the pull request itself or the proposed shift in how we should conduct notes. When I said "generally", I believe that I was just a little unsure whether or not it was worth the effort to go back and modify five weeks' worth of notes. Your point above that these notes represents us and should thus be polished makes sense to me and addresses that uncertainty, but in that case how far back would we need to look to ensure that our club is being represented well? I suppose the more recent ones are fine and are likely the only ones that really have a chance of any traffic.

**Comment by paulshriner** [comment]
> Your point above that these notes represents us and should thus be polished makes sense to me and addresses that uncertainty

I should clarify that my point was never to make the notes fully polished or nitpick every small detail. That would take way too much effort, and is not really appropriate for this situation (they are informal meeting notes, not an English paper). My point was that with the recent summer meeting notes, I believed that they were far too wordy, and like a wall of text. That makes it hard for someone reading to quickly find out the important details from the meeting (they would have to read almost everything).

Obviously, writing notes will depend on what happened in the meeting, but ways to make notes more readable include subsections, bullet points, labels (like question/answer sections), and simply breaking up large text blocks into smaller blocks. Again, this is in terms of the overall notes. If there's a misspelled word or a slightly wordy sentence it's fine, they won't detract from the points in the notes. But if the entire notes are a big wall of text with little separation, it will detract from the points.

> When I said "generally", I believe that I was just a little unsure whether or not it was worth the effort to go back and modify five weeks' worth of notes.

> but in that case how far back would we need to look to ensure that our club is being represented well? I suppose the more recent ones are fine and are likely the only ones that really have a chance of any traffic.

These are good points. I don't think it'd be worth it to make major changes to previous notes, they are in the past and people are likely not looking at them as much. Also, as we go farther back, it gets harder to remember what the original intentions of the meetings were, making major changes difficult. Finally, it might actually be good to leave the previous notes as they were, as they show our progress as a club.

Since I am a bit of a perfectionist and it would bother me if I didn't, I did make small changes to the previous notes (spelling errors, broken links, etc).

**Comment by lavender-aa** [comment]
these changes look good to me, i'll merge since it's been a while
